drivers/SmartThings/zigbee-window-treatment/profiles/window-treatment-micronic.yml

```diff
@@ -0,0 +1,19 @@
+name: window-treatment-micronic
+components:
+- id: main
+  capabilities:
+  - id: windowShade
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Blind
+preferences:
+  - name: "invert"
+    title: "Invert"
+    description: "Invert curtain direction"
+    required: true
+    preferenceType: boolean
+    definition:
+      default: false
```

Comment:
We have a `reverse` preference used in the z-wave profiles that is pseudo-official and is likely to receive localization support soon that you could use instead here, though I notice you've pulled this from an existing profile. Just in case you decide to change, it would simply be:

```
preferences:
  - preferenceId: reverse
    explicit: true
```